src/Microsoft.Azure.WebJobs.Host/Config/ExtensionOptionsInfo.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using Microsoft.Azure.WebJobs.Hosting;
+using System;
+
+namespace Microsoft.Azure.WebJobs.Host.Config
+{
+    public class ExtensionOptionsInfo : IExtensionOptionsInfo
```

Comment:
Because the interface is public, the implementation type ExtensionOptionsInfo can remain internal, right? To reduce public surface area.

Reply:
Good point. Done.